### PR TITLE
Add Insights page

### PR DESCRIPTION
**Pull request by desruc**
This PR adds the base Insights page.

More pages will be coming in later PRs.

### CODE REVIEW

File: docs/insights/index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+title: Insights
+description: Insights gives you better visibility into your company's DevOps performance.
+position: 100
+---
+
+![The Overview page of Insights Reports](images/overview.png)
+
+DevOps insights (early access) in Octopus gives you better visibility into your company’s DevOps performance by surfacing the four key DORA metrics, so you can make more informed decisions on where to improve and celebrate your results. 
+
+There are two levels available for DevOps Insights: Project level insights are available to all customers, and Space level insights are available to Enterprise customers.
```
Comment (harrisonmeister) [inline docs/insights/index.md:11]: Just a suggestion, I think the overview page is better after the first paragraph.

```suggestion
DevOps insights (early access) in Octopus gives you better visibility into your company’s DevOps performance by surfacing the four key DORA metrics, so you can make more informed decisions on where to improve and celebrate your results. 

![The Overview page of Insights Reports](images/overview.png)

Two levels are available for DevOps Insights:
1. Project-level insights, available to all customers.
2. Space-level insights, available to customers with an [enterprise license](https://octopus.com/enterprise).
```

File: docs/insights/index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- Mean time to recovery (MTTR)
+
+*Operation*
+- Reliability
+
+Throughput metrics measure the health of your deployment pipeline, while the stability indicators help you understand the quality of your software and delivery pipeline. In addition to the four classic DORA metrics that measure software delivery performance, DORA added a new measure in 2021 for operational performance.
+
+## Octopus built-in DORA metrics with DevOps Insights
+
+Release 2022.3 adds out-of-the-box support for the following DevOps metrics:
```
Comment (harrisonmeister) [inline docs/insights/index.md:34]:
```suggestion
Octopus **2022.3** adds out-of-the-box support for the following DevOps metrics:
```

File: docs/insights/index.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+**Deployment Frequency**
+
+How many deployments occur to the target environment.
+
+**Mean Time to Recovery**
+
+How long it takes to recover a failed deployment with a subsequent successful deployment.
+
+> Some of these metrics differ slightly from the textbook DORA metrics given the data available.
```
Comment (harrisonmeister) [inline docs/insights/index.md:52]:
```suggestion
:::hint
Some of these metrics differ slightly from the textbook DORA metrics, given the data available.
:::
```

File: docs/insights/index.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+- Help introduce change with data and collaboration to make a business case
+- Share successes and learn from failures to continuously improve
+
+## Understand performance of your projects with Project level Insights
+
+![Project Insights Deployment Frequency](images/project.png)
+
+Project level insights are available as a new tab in every project so you can understand performance of your projects across Channels, Environments and Tenants. Each metric can be seen at a summary level and insights can also be filtered to time frames including last month, quarter and year, channels and environments, as well as being exported into CSV.
+
+Project level insights are available to all customers out-of-the-box meaning you don’t have to buy or subscribe to another tool. If you’re already a user, Octopus already has all the data it needs to help you uncover rich insights based on your deployment history.
```
Comment (harrisonmeister) [inline docs/insights/index.md:68]: Grammarly suggestions and a suggestion to move the image to after the paragraphs

```suggestion
Project level insights are available as a new tab in every project so you can understand the performance of your projects across Channels, Environments, and Tenants. Each metric can be seen at a summary level, and insights can also be filtered to time frames, including last month, quarter, year, channels, and environments, as well as being exported into CSV.

Project level insights are available to all customers out-of-the-box, meaning you don’t have to buy or subscribe to another tool. If you’re already a user, Octopus has all the data it needs to help you uncover rich insights based on your deployment history.

![Project Insights Deployment Frequency](images/project.png)
```

File: docs/insights/index.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+## Gain insight across projects and teams with the Space level insights
+
+This release also includes additional insights capabilities for Enterprise customers. For customers at larger companies, we have built additional capabilities that make it easier to gain insight using DORA metrics in larger multi-team, multi-site and multi-project scenarios.
```
Comment (harrisonmeister) [inline docs/insights/index.md:72]:
```suggestion
Octopus **2022.3** also includes additional insights capabilities for customers with an [enterprise license](https://octopus.com/enterprise). For customers at larger companies, we have built additional capabilities that make it easier to gain insight using DORA metrics in larger multi-team, multi-site and multi-project scenarios.
```

File: docs/insights/index.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+Space level insights are available via the Insights tab and provide actionable DORA metrics for more complex scenarios across projects, projects groups, environments, or tenants. This enables managers and decision makers to get far more insight into the DevOps performance of their organization in line with their business context, such as team, portfolio or platform. Space level insights:
```
Comment (harrisonmeister) [inline docs/insights/index.md:74]:
```suggestion
Space level insights are available via the Insights tab and provide actionable DORA metrics for more complex scenarios across projects, project groups, environments, or tenants. This enables managers and decision-makers to get far more insight into the DevOps performance of their organization in line with their business context, such as team, portfolio, or platform. Space level insights:
```

File: docs/insights/index.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+- Aggregate data across your space so you can compare and contrast metrics across projects to identify what is working and what isn’t
+- Inform better decision making: identify problems, track improvements and celebrate successes 
+- Help you quantify DevOps performance based on what is actually happening as shown by data
```
Comment (harrisonmeister) [inline docs/insights/index.md:78]:
```suggestion
- Aggregate data across your space so you can compare and contrast metrics across projects to identify what is working and what isn’t
- Inform better decision making: identify problems, track improvements, and celebrate successes 
- Help you quantify DevOps performance based on what is actually happening, as shown by data
```

File: docs/insights/index.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+With the Space level insights, you can build reports with the relevant data that matters to you, choosing only relevant projects, project groups, environments, tenants. View information at a high level or drill down in project specific data over time.
```
Comment (harrisonmeister) [inline docs/insights/index.md:80]:
```suggestion
With the Space level insights, you can build reports with the relevant data that matters to you, choosing only relevant projects, project groups, environments, and tenants. View information at a high level or drill down into project-specific data over time.
```

File: docs/insights/index.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+This also allows you to easily identify contributors to a performance trend you see at the aggregated level. It provides a breakdown of highest and lowest performing projects, and releases that may have a significant impact on the overall performance. This enables you to identify where things are improving or declining and take action based on that.
```
Comment (harrisonmeister) [inline docs/insights/index.md:82]:
```suggestion
This also allows you to easily identify contributors to a performance trend you see at the aggregated level. It provides a breakdown of the highest and lowest performing projects and releases that may significantly impact the overall performance. This enables you to identify where things are improving or declining and take action based on that.
```

File: docs/insights/index.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+
+> Users who have view permissions to Space-level Insights reports will see sanitized data on projects / environments they don’t have access to.
+
+Enterprise DevOps Insights (Space-level) are available as a part of the Octopus Deploy Enterprise tier.
+
+Contact our sales team for a demo and learn more about [Octopus Deploy pricing](https://octopus.com/pricing)
```
Comment (harrisonmeister) [inline docs/insights/index.md:88]: I'm unsure if there is a past precedent for adding a request for customers to contact sales for a demo or pricing links in the docs. That doesn't mean we can't though :smile:.

```suggestion
Space level DevOps insights are available to customers with an [enterprise license](https://octopus.com/enterprise).
```

**Review by harrisonmeister — COMMENTED** [review]
Made some minor suggestions around grammar, image placement, and wording around the enterprise tier. For the Service now pages, we talk about the feature being available to customers with an enterprise license, so I've tried to follow that (admittedly somewhat weak) "standard"

Would be interested in @robpearson opinion on that vs "enterprise tier"

**Comment by harrisonmeister** [comment]
Update: I've changed the suggestion from "enterprise license" to "enterprise subscription". See [this discussion](https://octopusdeploy.slack.com/archives/C02G7KXS17F/p1662090523447839) for more details

**Review by robpearson — APPROVED** [review]
This looks great. I think it can be merged ASAP! :)

**Pull request merged**